docs/build-on-linea/tooling/data-indexers/subquery.md

```diff
@@ -0,0 +1,26 @@
+# Intro
+
+SubQuery is a leading blockchain data indexer that provides developers with fast, flexible, universal, open source and decentralised APIs for web3 projects. SubQuery SDK allows developers to get rich indexed data and build intuitive and immersive decentralised applications in a faster and more efficient way. SubQuery supports 100+ ecosystems including Linea, Ethereum, Polygon, Polkadot, Algorand, NEAR, and Avalanche.
+
+Another one of SubQuery's competitive advantages is the ability to aggregate data not only within a chain but across multiple blockchains all within a single project. This allows the creation of feature-rich dashboard analytics and multi-chain block scanners.
+
+Other advantages include superior performance with multiple RPC endpoint configurations, multi-worker capabilities and a configurable caching architecture. To find out more, visit our [documentation](https://academy.subquery.network/).
```

Comment:
```suggestion
Other advantages include superior performance with multiple RPC endpoint configurations, multi-worker capabilities and a configurable caching architecture. 
```

Not sure the link is really needed here given it's repeated in the bulleted list immediately below.